### DISCUSSION TITLE
Permission denied when changing config accessing RTL via TOR

### DISCUSSION THREAD

**Discussion by ghost** [Ride-The-Lightning/RTL#1200] [General]
I installed RTL on an Ubuntu Server and also managed to successfully reach it from another machine via TOR - so everything seems normal. But when I am for example try to change some settings in the config, I get the following permission denied errors:

Feb 17 11:33:23 username node[4741]: [2/17/2023, 11:33:23 AM] ERROR: Common => Error in Channel Backup for Node Node 1: EACCES: permission denied, open 'home/username/lnd-backups/channel-all.bak'

Feb 17 11:33:45 usernamet node[4741]: [2/17/2023, 11:33:45 AM] ERROR: RTLConf => Update Node Settings Error: {"statusCode":500,"message":"Update Node Settings Error","error":{"errno":-13,"syscall":"open","code":"EACCES","path":"/home/username/RTL-0.13.4/RTL-Config.json"}}

It looks like a problem with permissions (write access) to the config file or so - maybe can anyone help me on that?

**Comment by saubyk** [comment]
Hi @zyklopisch need more details to be able to advise any further action.
E.g.
- Where are you observing these error?
- What command are trying to run?
- What's your node configuration?

**Reply by ghost** [reply]
In the meanwhile, I did a reinstall and it seems to be that it was related somehow to root rights.... but now it works - thx!
